src/transformers/models/auto/tokenization_auto.py

```diff
@@ -138,7 +138,7 @@
                     "CpmTokenizerFast" if is_tokenizers_available() else None,
                 ),
             ),
-            ("cpmant", ("CpmAntTokenizer", None)),
+            ("cpmant", ("CPMAntTokenizer", None)),
```

Comment:
Nope, the actual tokenizer class name is `CpmAntTokenizer`, see https://github.com/huggingface/transformers/blob/5e099d64dec735576bfed3bf006c21412cef25d6/src/transformers/models/cpmant/tokenization_cpmant.py#L88

Reply:
ok😅, so should we modify the config of transformers or huggingface?@zh-zheng
https://huggingface.co/openbmb/cpm-ant-10b/blob/main/config.json

Reply:
sorry for the late reply but the model on the hub should be updated I believe!